managed_vms/datastore/src/main/java/com/example/managedvms/datastore/DatastoreServlet.java

```diff
@@ -44,17 +46,25 @@
   @Override
   public void doGet(HttpServletRequest req, HttpServletResponse resp) throws IOException,
       ServletException {
+    // encrypt user ip using PBEWithMD5AndDES
+    BasicTextEncryptor encryptor = new BasicTextEncryptor();
+    encryptor.setPassword(Double.toString(10000*Math.random()));
```

Comment:
I'm confused. If we're getting a new password every time, we're basically just storing a random number, not an IP address that we could recover.